src/Compiler/TaskModel.cpp

```diff
@@ -80,19 +80,21 @@ int TaskModel::columnCount(const QModelIndex &parent) const {
 }
 
 QVariant TaskModel::data(const QModelIndex &index, int role) const {
-  if (role == TaskId) return ToTaskId(index);
+  auto taskID = ToTaskId(index);
+  if (role == TaskId) return taskID;
 
-  auto task = m_taskManager->task(ToTaskId(index));
+  auto task = m_taskManager->task(taskID);
   if (!task) return QVariant();
 
   if (index.column() == TITLE_COL && role == Qt::CheckStateRole &&
       task->type() == TaskType::Action)
     return task->isEnable() ? Qt::Checked : Qt::Unchecked;
 
   if (role == Qt::DisplayRole && index.column() == TIMING_COL) {
-    auto registry = m_taskManager->getReportManagerRegistry().getReportManager(
-        ToTaskId(index));
-    if (registry) {
+    auto registry =
+        m_taskManager->getReportManagerRegistry().getReportManager(taskID);
+    // Bitstream generation step should not display Fmax
```

Comment:
@ecchg @guanyung-rs do we have any stats available (e.g. compression ratio) in logs that we can display for the user after bitstream generation ?